src/selectors/availableViewportWidth.js

```diff
@@ -18,7 +19,10 @@ import { createSelector } from 'reselect';
  * @return {number} The width available for rendering cells.
  */
 export default createSelector([
+  scrollbarsVisible,
   state => state.width,
-], width => {
-  return width - Scrollbar.SIZE - Scrollbar.OFFSET;
+], ({ scrollsVertically }, width) => {
+  const scrollbarSize = Scrollbar.SIZE + Scrollbar.OFFSET;
```

Comment:
Here's the naive fix.  If only the joint dependency didn't exist and cause all the data flow headaches...